slp2mp4/dolphinrunner.py

```diff
@@ -61,6 +62,29 @@ def count_frames_completed(self):
         print("Rendered ",num_completed," frames")
         return num_completed
 
+    def prep_dolphin_settings(self):
+        # Determine efb_scale value from resolution field in config
+        if self.conf.resolution not in RESOLUTION_DICT:
+            print("WARNING: configured resolution is not valid, using 480p")
+            efb_scale = RESOLUTION_DICT["480p"]
+        else:
+            efb_scale = RESOLUTION_DICT[self.conf.resolution]
+
+        # Apply graphics settings
+        gfx_ini = os.path.join(self.user_dir, "Config", "GFX.ini")
+        gfx_ini_parser = configparser.ConfigParser()
+        gfx_ini_parser.optionxform = str
+        gfx_ini_parser.read(gfx_ini)
+        gfx_ini_parser.set('Settings', 'EFBScale', efb_scale)
+        if self.conf.widescreen:
+            gfx_ini_parser.set('Settings', 'AspectRatio', "6")
+            with open(os.path.join(self.user_dir, "GameSettings", "GALE01.ini"), "a") as game_settings_file:
+                game_settings_file.write("\n$Widescreen 16:9")
```

Comment:
I decided to just append this to the file to enable the gecko code instead of using configparser because configparser didn't like '$'s.